scripts/deblur

```diff
@@ -68,18 +68,15 @@ def deblur_seqs(seqs_fp, read_error, mean_error, error_dist, indel_prob,
                 type=click.Path(resolve_path=True, readable=True, exists=True,
                                 file_okay=True))
 @click.argument('output_fp', required=True,
-                type=click.Path(resolve_path=True, readable=True, exists=True,
+                type=click.Path(resolve_path=True, readable=True, exists=False,
                                 file_okay=True))
 @click.option('--trim-length', '-t', required=False, type=int, default=100,
               help="Sequence trim length")
```

Comment:
add 'show_default=True' to @click.option() so that the default value is displayed during help ..